aries_cloudagent/messaging/valid.py

```diff
@@ -349,6 +349,27 @@ def __init__(self):
         )
 
 
+class RoutingKey(Regexp):
```

Comment:
I think this does the same thing as `IndyOrKeyDID`?

Reply:
`IndyOrKeyDID` looks for either an `IndyDID` or a `DIDKey`. In this instance, `RoutingKey` looks for either a `IndyRawPublicKey` or a `DIDKey`. A better name for it (based upon your question) might be `IndyRawPublicKeyOrDIDKey`. The only point of the class is to handle the migration from using `IndyRawPublicKey` during routing in messages and using `DIDKey` instead

Reply:
Ah yes, of course, thanks!